src/Symfony/Component/Form/Command/DebugCommand.php

```diff
@@ -195,13 +195,13 @@ private function getFqcnTypeClass(InputInterface $input, SymfonyStyle $io, strin
         return $io->choice(sprintf("The type \"%s\" is ambiguous.\n\nSelect one of the following form types to display its information:", $shortClassName), $classes, $classes[0]);
     }
 
-    private function getCoreTypes()
+    private function getCoreTypes(): array
     {
         $coreExtension = new CoreExtension();
         $loadTypesRefMethod = (new \ReflectionObject($coreExtension))->getMethod('loadTypes');
         $loadTypesRefMethod->setAccessible(true);
         $coreTypes = $loadTypesRefMethod->invoke($coreExtension);
-        $coreTypes = array_map(function (FormTypeInterface $type) { return \get_class($type); }, $coreTypes);
+        $coreTypes = array_map(function (FormTypeInterface $type): string { return \get_class($type); }, $coreTypes);
```

Comment:
I don't think adding these to internal closures provide anything

Reply:
Reverted. TBH, the whole closure feels unnecessary. Why aren't we using `'get_class'` as callable here? 🤔